src/chpasswd.c

```diff
@@ -554,6 +555,21 @@ int main (int argc, char **argv)
 		} else
 #endif				/* USE_PAM */
 		{
+
+		/*
+		 * Prevent adding a non valid hash to /etc/shadow and
+		 * potentialy lock account
+		 */
+
+		if (eflg) {
+			if (!is_valid_hash(newpwd)) {
+				fprintf (stderr,
+					_("%s: (line: %jd) invalid password hash for user '%s'\n"),
```

Comment:
How about `_("%s:%d: invalid ...\n")`, which is more compact?

Reply:
Ahh, sorry, I see that other diagnostics say line:.  So let's keep that for consistency.